File: 02. C++/08. Operadores lógicos/Operadores logicos.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{

    /* Y = && = Las dos proposiciones deben ser verdaderas*/
    cout<<((true && true))<<endl; //1
    cout<<((true && false))<<endl; //0
    cout<<((false && true))<<endl; //0
    cout<<((false && false))<<endl; //0

    /* O = || = Una de las proposiciones debe ser verdadera*/
    cout<<((true || true))<<endl; //1
    cout<<((true || false))<<endl; //1
    cout<<((false || true))<<endl; //1
    cout<<((false || false))<<endl; //0

    /* NO = ! = Niega el valor de verdad */
    cout<<(!(true))<<endl; //0
    cout<<(!(false))<<endl; //1
	
}

File: 05. Lenguajes no clasificados/02. Visual Basic/08. Operadores lógicos/Operadores logicos.vb
Module Module1

    Sub Main()

        'Y = And = Las dos proposiciones deben ser verdaderas
        Console.WriteLine((True And True)) 'true
        Console.WriteLine((True And False)) 'false
        Console.WriteLine((False And True)) 'false
        Console.WriteLine((False And False)) 'false

        'O = Or = Una de las proposiciones debe ser verdadera
        Console.WriteLine((True Or True)) 'true
        Console.WriteLine((True Or False)) 'true
        Console.WriteLine((False Or True)) 'true
        Console.WriteLine((False Or False)) 'false

        'NO = Not = Niega el valor de verdad
        Console.WriteLine(Not (True)) 'false
        Console.WriteLine(Not (False)) 'true

        Console.In.ReadLine()

    End Sub

End Module
